auth-tokens/src/test/java/com/palantir/tokens/auth/UnverifiedJsonWebTokenTests.java

```diff
@@ -90,4 +98,10 @@ public void invalidJwt_invalidPayloadToken() {
             assertEquals("Invalid JWT: cannot parse payload", e.getMessage());
         }
     }
+
+    private void assertValidToken(UnverifiedJsonWebToken token) {
```

Comment:
these tests aren't just asserting that the token's valid - they're asserting facts about what's in the token. For example, the session token test is asserting that the token is valid, too.

This method should be called 'assertValidApiToken'